components/_patterns/01-atoms/08-video/_video.scss

```diff
@@ -1,11 +1,17 @@
-// Responsive Video using only CSS
+// Responsive Video using CSS only
 .video {
   height: 0;
   overflow: hidden;
-  padding-bottom: 56.25%;
   padding-top: 35px;
   position: relative;
 
+  &--wide {
+    padding-bottom: 56.25%; // 56.25% = 16x9
```

Comment:
@evanmwillhite should we maybe move this back up into the .video selector so that it defaults to support 16x9? and keep the &--full modifier so that it can be overridden in special cases?